src/bci_build/package/helm/README.md.j2

```diff
@@ -24,10 +28,10 @@ For a comprehensive guide on getting started with Helm, refer to the [official H
 
 ### Connecting Helm container to the Host's Kubernetes Cluster
 
-To interact with a Kubernetes cluster running on the host, mount the Kubernetes configuration file (`kubeconfig`) into the container (use `--net=host` flag to allow container to use host’s network):
+To interact with a Kubernetes cluster running on the host, mount the Kubernetes configuration file (`kubeconfig`) into the container (use the `--net=host` flag to allow the container to use the host’s network):
```

Comment:
In a production scenario, you will never execute Helm on the host (aka control plane), always remote (dev machine).

The kubeconfig is not from the host cluster, it is from a dev machine.

> To interact with a Kubernetes cluster, mount the Kubernetes configuration file (`kubeconfig`) from the container host into the container (use the `--net=host` flag to allow the container to use the host’s network):

Reply:
I like the rewording suggestion, it makes it a lot clearer. There is a 2nd use case, to have a admin privileged pod running within the kubernetes cluster. this was the original request for this container. see e.g.

https://serverfault.com/questions/1013420/kubernetes-kubeconfig-inside-pod
https://stackoverflow.com/questions/72874454/how-to-access-kubeconfig-file-inside-containers

and similar. fact is we could be supporting both usecases here.